src/Components/Style/Recently-played-zone.css

```
@@ -0,0 +1,108 @@
+@import "./Colors.css";
+@import "./Fonts.css";
+
+.recently-played-zone-container {
+  align-items: center;
+  align-self: center;
+  background-color: var(--recently-played-zone-shadow-color);
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 10px;
+  width: fit-content;
+}
+
+.recently-played-label {
+  color: var(--font-color);
+  font-family: var(--card-font);
+  height: fit-content;
+  margin: 10px;
+}
+
+.recently-played-cards-container-0 {
+  display: none;
+}
+
+.recently-played-cards-container-1,
+.recently-played-cards-container-2,
+.recently-played-cards-container-3 {
+  align-items: center;
+  display: flex;
+  height: 95%;
+  justify-content: center;
+  margin: 10px;
+  width: 95%;
+}
+
+.recently-played-cards-container-2 {
+  justify-content: space-evenly;
+}
+
+.recently-played-cards-container-3 {
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 380px;
+}
+
+.recently-played-cards-container-4 {
+  column-gap: 5px;
+  display: grid;
+  grid-template-columns: 190px 190px;
+  grid-template-rows: 230px 230px;
+  height: fit-content;
+  justify-items: center;
+  margin: 10px;
+  row-gap: 0px;
+}
+
+.recently-played-cards-container-0 > .card,
+.recently-played-cards-container-1 > .card,
+.recently-played-cards-container-2 > .card,
+.recently-played-cards-container-3 > .card,
+.recently-played-cards-container-4 > .card {
+  height: 206px;
+  margin: 10px;
+  width: 170px;
+}
```

when having the same CSS rule for numbered containers or similar stuff, you can use a class wildcard like this
```suggestion
div[class^="recently-played-cards-container-"] > .card {
  height: 206px;
  margin: 10px;
  width: 170px;
}
```
this is checking if the class attribute starts with the string you provided, so the number at the end can be dynamic

---

the same comment applies to other instances on this CSS, you can find an explanation here: https://www.geeksforgeeks.org/wildcard-selectors-and-in-css-for-classes/